### ISSUE TITLE
lh-cpp: broke visual mode '<'

### ISSUE THREAD

**Issue by GoogleCodeExporter** [byzhang/lh-vim#5]
```
lh-cpp broke visual mode '<'. Is there a way to
disable your functionality (surrounding with <>)?

```

Original issue reported on code.google.com by `ngaloppo@gmail.com` on 26 Feb 2008 at 8:57

**Comment by GoogleCodeExporter** [comment]
```
The correction of this defect will be based on the corrections of the issues #3 
and #4.
```

Original comment by `luc.herm...@gmail.com` on 3 Mar 2008 at 4:16
- Changed title: **lh-cpp: broke visual mode '<'**
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Corrected thanks to the new kernel of map-tools v1.0.0.
```

Original comment by `luc.herm...@gmail.com` on 25 Mar 2008 at 2:27
- Changed state: **Fixed**
